src/test/java/ca/gc/aafc/collection/api/entities/GeoreferenceAssertionCRUDIT.java

```diff
@@ -47,7 +50,6 @@ public void testFind() {
         .dwcDecimalLongitude(45.01)
         .dwcCoordinateUncertaintyInMeters(10)
         .dwcGeoreferencedDate(testGeoreferencedDate)
-        .dwcGeoreferenceVerificationStatus(georeferenceVerificationStatus)
         .build();
```

Comment:
not sure why we are removing this from the test case, this should be a valid test of the entity

Reply:
additionally georeferenceVerificationStatus is never used after these changes